mockgcp/mockgcptests/proxy.go

```diff
@@ -121,6 +121,8 @@ func (p *Proxy) BuildGcloudConfig(proxyEndpoint *net.TCPAddr, mockgcp mockgcp.In
 		"storage.googleapis.com",
 		"cloudresourcemanager.googleapis.com",
 		"serviceusage.googleapis.com",
+		"workflows.googleapis.com",
+		"workflowexecutions.googleapis.com",
```

Comment:
Cool! Is this added by AI via some hints like "update the service in proxy.go" or manually added?

Reply:
I think I manually added these, but yes it would be easy enough to ask AI to add them automatically.  I guess one gotcha is that we don't actually know the service names at this stage (until we've run the test).

Maybe we should just add all the hostnames we know of (or at least the ones that gcloud knows of!)

There might also be a better way to intercept gcloud traffic, I just haven't found one yet!